src/components/metrics/CountCard.jsx

```diff
@@ -1,20 +1,34 @@
 import React from "react"
 import PropTypes from "prop-types"
+import _ from "lodash"
 
-const CountCard = ({ title, count }) => (
+const CountCard = ({ title, count = null, footer = null }) => (
   <div className="card">
     <div className="card-header">
       <h5>{title}</h5>
     </div>
     <div className="card-body">
-      <p className="card-text display-1 text-center">{count}</p>
+      <div className="card-text display-1 text-center">
+        {_.isNull(count) ? (
+          <div className="spinner-border" role="status">
+            <span className="visually-hidden">Loading...</span>
+          </div>
+        ) : (
+          <React.Fragment>{count}</React.Fragment>
+        )}
+      </div>
     </div>
+    {footer && <div className="card-footer">{footer}</div>}
   </div>
 )
 
 CountCard.propTypes = {
   title: PropTypes.string.isRequired,
-  count: PropTypes.number.isRequired,
+  count: PropTypes.number,
+  footer: PropTypes.oneOfType([
+    PropTypes.node,
+    PropTypes.arrayOf(PropTypes.node),
+  ]),
```

Comment:
Can it be one or the other, currently? Below, I'm only seeing fragment instances being provided as footers but maybe I'm overlooking something. Or maybe this is future-proofing?

Reply:
Per Google, this is the recommended approach for this sort of property, so yeah, future-proofing.

Reply:
Also, this is used elsewhere in our code, so this is consistent.

Reply:
🍻